### ISSUE TITLE
Double backslash in highlights folder name

### ISSUE THREAD

**Issue by 54656452** [instaloader/instaloader#560]
**Describe the bug**
I am unable to download highlights - possibly due to instaloader trying to due a double backslash in the path for reasons I have not determined. Other highlights from the profile download without issue.

**To Reproduce**
1. Follow @dwarvenforgeofficial
2. `instaloader --fast-update --commit-mode --geotags --stories --highlights --igtv --login <username>`

**Expected behavior**
These highlights to download successfully like those in the KS7 Brainstorm highlights did.

**Error messages and tracebacks**
```
[ 11/ 11] dwarvenforgeofficial﹨KS7 Brainstorm/2019-08-21_21-14-51_UTC.jpg dwarvenforgeofficial﹨KS7 Brainstorm/2019-08-21_21-14-51_UTC.mp4 json
Retrieving highlights "Where’s Nate?" from profile dwarvenforgeofficial
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.8_3.8.752.0_x64__qbz5n2kfra8p0\lib\runpy.py", line 193, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.8_3.8.752.0_x64__qbz5n2kfra8p0\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Media\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\Scripts\instaloader.exe\__main__.py", line 7, in <module>
  File "C:\Users\Media\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\instaloader\__main__.py", line 412, in main
    _main(loader,
  File "C:\Users\Media\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\instaloader\__main__.py", line 192, in _main
    instaloader.download_profiles(profiles,
  File "C:\Users\Media\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\instaloader\instaloader.py", line 1082, in download_profiles
    self.download_highlights(profile, fast_update=fast_update, storyitem_filter=storyitem_filter)
  File "C:\Users\Media\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\instaloader\instaloader.py", line 49, in call
    return func(instaloader, *args, **kwargs)
  File "C:\Users\Media\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\instaloader\instaloader.py", line 679, in download_highlights
    self.download_highlight_cover(user_highlight, highlight_target)
  File "C:\Users\Media\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\instaloader\instaloader.py", line 409, in download_highlight_cover
    self.download_title_pic(highlight.cover_url, target, 'cover', highlight.owner_profile)
  File "C:\Users\Media\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\instaloader\instaloader.py", line 65, in call
    return func(instaloader, *args, **kwargs)
  File "C:\Users\Media\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\instaloader\instaloader.py", line 395, in download_title_pic
    os.makedirs(os.path.dirname(filename), exist_ok=True)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.8_3.8.752.0_x64__qbz5n2kfra8p0\lib\os.py", line 223, in makedirs
    mkdir(name, mode)
OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'dwarvenforgeofficial\\Where’s Nate?'
```

**Instaloader version**
4.3.1

**Additional context**

**Comment by i-cant-git** [comment]
Same problem but with character `...`.

It errors because it doesn't use `_PostPathFormatter` like methods that use `Post`.

I don't know the difference between `vformat` and `get_value` but `..` and `...` should be replaced with below:

```py
ret = ret.replace('...', '\u2026').replace('..', '\u2025')
```

* `‥` = `\u2025` http://www.fileformat.info/info/unicode/char/2025/index.htm
* `…` = `\u2026` http://www.fileformat.info/info/unicode/char/2026/index.htm

**Comment by aandergr** [comment]
Has this been resolved with [Version 4.4.3](https://github.com/instaloader/instaloader/releases/tag/v4.4.3)? I just tried downloading the highlights of the exact profile on Windows 10 and it worked without any problems.
